cmd/testcase_validate.go

```diff
@@ -36,6 +38,10 @@ Examples
 
 	forge test-case validate acme-inc ./dist/foo.js ./dist/bar.js ./dist/foobar.js
 
+Bundling
+--------
+
+Validate automatically bundles your javascript file, if you use the .mjs extension. See 'forge build' for more details.
```

Comment:
Does it make sense to extract this usage hints into a variable? So that it stays consistent everywhere?

Reply:
Thats a good idea 👍

Reply:
But the first word is different for each command ..

Reply:
hmm, true. Maybe we can rephrase that a bit. Let's see.